src/test/java/uk/gov/hmcts/cmc/claimstore/events/paindinfull/PaidInFullCitizenNotificationHandlerTest.java

```diff
@@ -42,17 +45,44 @@ public void setUp() {
     }
 
     @Test
-    public void sendNotificationsSendsNotificationsToDefendant() throws NotificationClientException {
-
-        PaidInFullEvent event = new PaidInFullEvent(SampleClaim.getDefault());
+    public void sendNotificationsSendsNotificationsToDefendantEmailNotLinked() throws NotificationClientException {
+        Response fullDefenceResponse = SampleResponse.FullDefence.builder().build();
+        PaidInFullEvent event = new PaidInFullEvent(SampleClaim.getWithResponse(fullDefenceResponse));
 
         handler.notifyDefendantForPaidInFull(event);
+        verify(notificationService, once()).sendMail(
+            eq(event.getClaim().getDefendantEmail()),
```

Comment:
Ad defendant is not linked, this should be email provided claimant i.e. `claim.getClaimData().getDefendant().getEmail()`